Builder/inc/CodeBuilder.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef CODEBUILDER_H
+#define CODEBUILDER_H
+
+#include <string>
+#include <ostream>
+#include <vector>
+
+using namespace std;
+
+struct Field
+{
+    string name{}, type{};
```

Comment:
Why are struct fields public?

Reply:
There was no apparent reason behind it. I've changed the struct to a class and made the fields private.